scl-rubygem-GEM.spec.template

```diff
@@ -15,6 +15,11 @@ URL:     <%= spec.homepage %>
 <% end -%>
 Source0: <%= download_path %>%{gem_name}-%{version}.gem
 
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+<% for d in spec.runtime_dependencies -%>
+Requires: %{?scl_prefix}rubygem(d.name)
```

Comment:
Shouldn't this include the version requirement too?

Reply:
Take a look now at the template. I attempted my best at updating it to account for those special cases.